doc/classes/GraphElement.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GraphElement" inherits="Container" is_experimental="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		GraphElement is a [Container] control that provides the basic functionality to be used inside a [GraphEdit] control (as a child), meaning it will be selectable and transformable by the graph editor.
+	</brief_description>
+	<description>
+		GraphElement allows to create your own custom nodes which can be used as a child of a [GraphEdit] node, but they won't be able to connect to other nodes.
+		To create a custom node that can connect to other nodes, extend [GraphNode].
+	</description>
+	<tutorials>
+	</tutorials>
+	<members>
+		<member name="draggable" type="bool" setter="set_draggable" getter="is_draggable" default="true">
+			If [code]true[/code], the user can drag the GraphElement.
+		</member>
+		<member name="position_offset" type="Vector2" setter="set_position_offset" getter="get_position_offset" default="Vector2(0, 0)">
+			The offset of the GraphElement, relative to the scroll offset of the [GraphEdit].
+			[b]Note:[/b] You cannot use the position offset directly, as [GraphEdit] is a [Container].
```

Comment:
```suggestion
			[b]Note:[/b] You cannot use the position offset directly, as [GraphEdit] is a [Container].
```

This comes from the existing description, but this is wrong and confusing. GraphEdit is not a container, and I'm not entirely sure what is being said here. What kind of "use" is implied. So I'd suggest removing the line, unless you know what it means, then we can improve it.

Reply:
I'm confused as well about this sentence. Let's remove it.